### PR TITLE
Portfolio:  Set up and mobile first

### PR DESCRIPTION
**Pull request by ikennarichard**
## Module 1: Portfolio set up and mobile first
For this project, I:
- Created a portfolio homepage from a template on Figma
- Implemented a mobile first approach
- Added a header section
- Added a headline section, which includes a heading and a summary text
- I used linters and GitHub actions to check for inconsistencies
- Made the page responsive using media queries and relative units

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        <div class="social_links">
+            <img src="./icons/twitter.svg" alt="twitter icon">
+            <img src="./icons/Linkedin icon.svg" alt="linked icon">
+            <img src="./icons/medium.svg" alt="medium icon">
+            <img src="./icons/github.svg" alt="github icon">
+            <img src="./icons/angellist.svg" alt="angellist icon">
+        </div>
+
+        <button class="next_button"><img src="./icons/arrow-icon.svg" alt="down_arrow_icon"></button>
+    </main>
```
Comment (AlphaNtihinduka) [inline index.html:30]: - [ ] You have done a great job so far 👏 . However, remember we are looking at ensuring that our portfolio is more professional ☺️ . Therefore I suggest that the container of the section is a `<section>` tag. This ensures the easy accessibility of the project to the screen readers which will help people with poor sight 😎  to navigate through your portfolio.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    <link rel="stylesheet" href="styles.css">
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Ikenna Richard</title>
+</head>
+<body>
+    <main> 
+        <header> <!-- work on header section -->
+            <h3>Welcome</h3>
```
Comment (AlphaNtihinduka) [inline index.html:13]: - [ ] If you notice, most of the websites have a logo, such that when the user clicks, the page is redirected to the home page ☺️ . This means the logo is always `a` tag. Therefore, I suggest that you enclose the logo in the link. In addition, `h1` tag should be used once on the page and used only on headings mostly found in the banner.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+        <p class="headline_paragraph"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, <br>don’t hestiate to contact me.</p>
+
+        <div class="social_links">
+            <img src="./icons/twitter.svg" alt="twitter icon">
+            <img src="./icons/Linkedin icon.svg" alt="linked icon">
+            <img src="./icons/medium.svg" alt="medium icon">
+            <img src="./icons/github.svg" alt="github icon">
+            <img src="./icons/angellist.svg" alt="angellist icon">
+        </div>
```
Comment (AlphaNtihinduka) [inline index.html:27]: - [ ] We are now getting close to complete this section 🚶 . Let's bring back our focus to the social links now.  The list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag). This helps us to ensure that the HTML tags are [Semantically](https://www.w3schools.com/html/html5_semantic_elements.asp) used ☺️ . In addition, this ensures the easy accessibility of the project to the screen readers which will help people with poor sight 😎 to navigate through your portfolio.

File: styles.css [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+  --crayola: #acb7c3;
+  --platinum: #dfe1e6;
+}
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+
+  /* border: 1px solid; */
```
Comment (AlphaNtihinduka) [inline styles.css:18]: - [ ] I am excited that you ensured the use of comments in your code 👏 . Comments are program text used to explain the program logic. They are ignored by the compiler. Comments help to make our code more readable and maintainable. However, we should ensure that we don't leave the code commented. This makes your codes messier and therefore they should be deleted 😞 .

**Review by AlphaNtihinduka — CHANGES_REQUESTED** [review]
## Hi @ikennarichard 🙋‍♂️
Good job working on your first contact form. You have done the design to the best expectation and incredibly well. 👏  

However, there are some issues mostly on the the semantic HTML 😞 .

You still need to implement few on the project but you are almost there!!! 🚶  

TO HIGHLIGHT:
- Your project is professional ✅
- Professional README file :100: 
- Your PR is professional ✅
- Well-structured files ✅

# STATUS: Required Changes :recycle: 

close

## Required changes ♻️ 
Kindly ensure to include the inline comments.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or slack channel if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK :crossed_fingers: :four_leaf_clover:

File: styles.css [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+  padding: 0;
+}
+
+body {
+  display: flex;
+  flex-direction: column;
+  place-items: center;
+  min-height: 100vh;
+  font-family: 'Inter', sans-serif;
+}
```
Comment (iambenkis) [inline styles.css:23]: - [ ] Good design :+1: , keep doing the great work

**Review by iambenkis — APPROVED** [review]
# Project Approved :trophy: :tada: 🟢 

Hi @ikennarichard  

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

Your project is professional  :heavy_check_mark: 
Professional README file  :heavy_check_mark: 
Your PR is professional  :heavy_check_mark: 
Well-structured files  :heavy_check_mark: 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@iambenkis** in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U02FTTCCU2V)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**